hyperopt/tests/test_fmin.py

```diff
@@ -291,3 +291,42 @@ def test_invalid_timeout():
             )
         except Exception as e:
             assert str(e) == expected_message
+
+
+def test_loss_threshold():
+    loss_threshold=0.001
+    hypopt_trials = Trials()
+    best = fmin(
+        fn=lambda x: x ** 2,
+        space=hp.uniform('x', -10, 10),
+        loss_threshold=loss_threshold,
+        algo=rand.suggest,
+        trials=hypopt_trials,
+        return_argmin=False,
+        #max_evals=100
```

Comment:
please set a rstate so tests are (more) deterministic, and remove the commented line if not needed